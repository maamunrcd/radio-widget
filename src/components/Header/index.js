import React from "react";
import "./index.scss";
import arrIcon from "../../assets/images/arrow.png";
import switchIcon from "../../assets/images/switch.png";

export default function Header() {
  return (
    <div className="header">
      <span className="icon-arrow">
        <img src={arrIcon} alt="arrow icon" />
      </span>
      <h1 className="title">STATIONS</h1>
      <span className="icon-power">
        <img src={switchIcon} alt="power icon" />
      </span>
    </div>
  );
}
