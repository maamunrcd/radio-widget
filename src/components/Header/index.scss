@import '../../assets/scss/variable.scss';

.header {
    height: 15%;
    background: $header-color;
    border-top-left-radius: 45px;
    border-top-right-radius: 45px;
    padding: 40px;
    display: flex;
    justify-content: space-between;
    align-items: center;
    font-weight: 600;
}