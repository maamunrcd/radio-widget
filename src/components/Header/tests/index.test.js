import React from "react";
import { render } from "@testing-library/react";
import Header from "../index";

describe("<Header />", () => {
  it("should render its heading", () => {
    const {
      container: { firstChild },
    } = render(<Header />);

    expect(firstChild).toMatchSnapshot();
  });
});
