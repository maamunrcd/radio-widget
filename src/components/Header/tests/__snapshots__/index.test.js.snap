// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render its heading 1`] = `
<div
  class="header"
>
  <span
    class="icon-arrow"
  >
    <img
      alt="arrow icon"
      src="arrow.png"
    />
  </span>
  <h1
    class="title"
  >
    STATIONS
  </h1>
  <span
    class="icon-power"
  >
    <img
      alt="power icon"
      src="switch.png"
    />
  </span>
</div>
`;
